CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [4.3.0](https://github.com/sonata-project/SonataFormatterBundle/compare/4.2.0...4.3.0) - 2020-09-24
+### Added
+- [[#477](https://github.com/sonata-project/SonataFormatterBundle/pull/477)] Added support for symfony/options-resolver:^5.1 ([@phansys](https://github.com/phansys))
+- [[#477](https://github.com/sonata-project/SonataFormatterBundle/pull/477)] Added support for symfony/property-access:^5.1 ([@phansys](https://github.com/phansys))
+
+### Changed
+- [[#472](https://github.com/sonata-project/SonataFormatterBundle/pull/472)] Allow FOSCKEditor dependency to ^2.0 ([@bmaziere](https://github.com/bmaziere))
```

Comment:
```suggestion
- [[#472](https://github.com/sonata-project/SonataFormatterBundle/pull/472)] Added support for friendsofsymfony/ckeditor-bundle:^2.0 ([@bmaziere](https://github.com/bmaziere))
```

Reply:
Even if we can, I think we should try to avoid editing the original changelog messages after a PR is merged.
If there is something to change, that suggestion must be done when the PR is open.
One thing would be a small change or a typo fix, but changing the syntax is something I think we mustn't do.

Reply:
I agree with it. IMO we should add some rules about Changelog, prefer using package name and avoid using repository name when we are using version.